src/ripple/app/tx/impl/XChainBridge.cpp

```diff
@@ -119,12 +119,24 @@ enum class DepositAuthPolicy { normal, dstCanBypass };
            transfer.
     @param depositAuthPolicy Flag to determine if dst can bypass deposit auth if
            it is also the claim owner.
+    @param submittingAccountInfo If the transaction is allowed to dip into the
+           reserve to pay fees, then this optional will be seated ("commit"
+           transactions support this, other transactions should not).
     @param j Log
 
     @return tesSUCCESS if payment succeeds, otherwise the error code for the
             failure reason.
  */
 
+// Allow the fee to dip into the reserve. To support this, information about the
```

Comment:
Please consider move the `struct TransferHelperSubmittingAccountInfo` up before the function document of `transferHelper()`. At least my IDE won't find the function document if it is separated from the function itself.